Oracle/PCORNetLoader_ora.sql

```diff
@@ -1335,7 +1275,8 @@ m.start_date SPECIMEN_DATE,
 to_char(m.start_date,'HH:MI')  SPECIMEN_TIME,
 m.end_date RESULT_DATE,
 to_char(m.end_date,'HH:MI') RESULT_TIME,
-CASE WHEN m.ValType_Cd='T' THEN NVL(nullif(m.TVal_Char,''),'NI') ELSE 'NI' END RESULT_QUAL, -- TODO: Should be a standardized value
+--CASE WHEN m.ValType_Cd='T' THEN NVL(nullif(m.TVal_Char,''),'NI') ELSE 'NI' END RESULT_QUAL, -- TODO: Should be a standardized value
+'NI' RESULT_QUAL, -- Temporary fix for KUMC

```

Comment:
@mprittie: Do you have a design in mind that will allow you to revert this temporary fix?